src/MoBi.Presentation/Presenter/TagsPresenter.cs

```diff
@@ -0,0 +1,64 @@
+﻿using MoBi.Presentation.DTO;
+using MoBi.Presentation.Mappers;
+using MoBi.Presentation.Presenter.BasePresenter;
+using MoBi.Presentation.Tasks;
+using MoBi.Presentation.Views;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Presentation.Presenters;
+using OSPSuite.Utility.Collections;
+using OSPSuite.Utility.Extensions;
+
+namespace MoBi.Presentation.Presenter
+{
+   //AppConstants.Captions.ContainerTags.FormatForLabel
+
+   public interface ITagsPresenter : IEditPresenter<IEntity>, IPresenter<ITagsView>
+   {
+      void AddNewTag();
+      void RemoveTag(TagDTO tagDTO);
+      IBuildingBlock BuildingBlock { get; set; }
+   }
+
+   public class TagsPresenter : AbstractEditPresenter<ITagsView, ITagsPresenter, IEntity>, ITagsPresenter
```

Comment:
Simple  implementation. Very easy to reason about, test in isolation etc.
We should try to do the same for all presenters function we will create in the future.